packages/augur-ui/src/modules/universe/actions/load-universe-details.ts

```diff
@@ -0,0 +1,19 @@
+import { augurSdk } from 'services/augursdk';
+import { ThunkAction } from 'redux-thunk';
+import { updateUniverse } from 'modules/universe/actions/update-universe';
+
+export const loadUniverseDetails = (
+  universe: string,
+  account: string
+): ThunkAction<any, any, any, any> => async (
+  dispatch,
+  getState
+) => {
+  const augur = augurSdk.get();
+  const universeDetails= await augur.getUniverseChildren
+  ({
+    universe,
+    account
+  });
+  dispatch(updateUniverse({ ...universeDetails }));
```

Comment:
Children of parent universe go in children collection of parent universe. 
The naming seems a bit confusing if this call is getting current univers details.

Reply:
Won't the children array that's in the state's universe get overwritten with the children in universeDetails? Are you just saying the naming should be different?

Reply:
I might be mistake on how the getter works. I see 

better name for the getter would be `getUniverseWithChildren` since it returns the parent with children. ok I get it. yea, that looks good.